src/screens/AddBikeStack/AddBikeSubmit.jsx

```diff
@@ -66,7 +66,7 @@ AddBikeSubmit.propTypes = {
           style: PropTypes.string.isRequired,
           frame: PropTypes.string.isRequired,
           keywords: PropTypes.string,
-          lock: PropTypes.number.isRequired
+          lock: PropTypes.string.isRequired
```

Comment:
The input component from the AddBikeForm is converting this into a string, even though the keyboard type is numeric. Since the keyboard type is numeric and the Yup number validation is working, we shouldn't have a problem with invalid values getting this far, so I see no problem with saving lock as a string to the DB.